addons/mail/static/src/core/web/activity_menu.js

```diff
@@ -102,3 +103,30 @@ export class ActivityMenu extends Component {
 registry
     .category("systray")
     .add("mail.activity_menu", { Component: ActivityMenu }, { sequence: 20 });
+
+const commandProviderRegistry = registry.category("command_provider");
+
+commandProviderRegistry.add("activity", {
```

Comment:
Should be in a command_palette.js file as far as I understand JS file oranization